css/main.css

```diff
@@ -52,9 +52,9 @@ nav .logo {
 }
 
 nav .user-greet {
-  display: none;
   font-size: 20px;
-  margin-top: 10px;
+  margin: 20px 30px 0 0 !important;
```

Comment:
Sorry for the late response, could we look into the way , where we don't use `!important` please